wellcomecollection/editorial-photography-ingest
# Upload

## What does this change?

This adds upload functionality to the application.  Currently, it is hard-coded to upload to the staging bucket.

## How to test

With a zip on your local filesystem, 

`AWS_TARGET_PROFILE=digitisation-developer PYTHONPATH=src python src/transferrer/upload.py path/to/zip`

Then have a look on s3: 
-  Using a profile that can see digitisation buckets
-  Have a look in the staging bucket s3://wellcomecollection-archivematica-staging-transfer-source/born-digital-accessions/
